### PR TITLE
#111 statuscode 400 in swagger and update guideline

### PR DESCRIPTION
**Pull request by pietersap**
#111 

Updated guideline and added 400 status code in swagger. I actually did this both in maturity level 1 and 2.

### CODE REVIEW

File: maturity-level-one/README.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ This allows you to be secure on the physical API while the consumers can still u
     - Less details compared to a custom data contract
     - Read [this blog post](https://tech.domain.com.au/2017/11/please-dont-spare-me-the-details/) on how to achieve this
   - If the above suggestion is not possible you should use a custom data contract. See Microsoft example [here](https://github.com/Microsoft/api-guidelines/blob/master/Guidelines.md#7102-error-condition-responses).
+- For all controllers, document all possible return status codes using a [SwaggerResponse] attribute.
```
Comment (MassimoC) [inline maturity-level-one/README.md]: Can you put this comment in the "Generating OpenAPI documentation" table ?
There is a row with SwaggerResponse attribute

Comment (pietersap) [reply maturity-level-one/README.md]: Fixed

**Comment by tomkerkhove** [comment]
Any update on this @pietersap 	@MassimoC 	?

**Comment by tomkerkhove** [comment]
Do you have some time to finish this PR @pietersap 	?

**Comment by tomkerkhove** [comment]
Thx!

**Pull request merged**